templates/email/ChangePasswordEmail.ss

```diff
@@ -2,10 +2,6 @@
 
 <p>
 	<%t ChangePasswordEmail_ss.CHANGEPASSWORDTEXT1 'You changed your password for' is 'for a url' %> $AbsoluteBaseURL.<br />
-	<%t ChangePasswordEmail_ss.CHANGEPASSWORDTEXT2 'You can now use the following credentials to log in:' %>
-</p>
-
-<p>
-	<%t ChangePasswordEmail_ss.EMAIL 'Email' %>: $Email<br />
-	<%t ChangePasswordEmail_ss.PASSWORD 'Password' %>: $CleartextPassword
+	<%t ChangePasswordEmail_ss.CHANGEPASSWORDTEXT2 'The password for account with email address {email} has been changed. If you didn't change your password please change your password using the link below' email=$Email %><br />
```

Comment:
Please don't re-define translation strings re-using old keys; You'll need to create a new translation string instead.

All translations are shared across all versions, so this would cause this string to appear on all 3.x releases, unfortunately.